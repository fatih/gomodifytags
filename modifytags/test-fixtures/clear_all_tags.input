package foo

type foo struct {
	bar string `json:"bar,omitempty" hcl:"bar,omitnested"`
	t   bool   `hcl:"t"`
	t   bool   `hcl:"t,omitempty"`
	qux string `json:"qux,omitempty" hcl:"qux,squash,keys"`
	yoo string `json:"yoo"`
}
