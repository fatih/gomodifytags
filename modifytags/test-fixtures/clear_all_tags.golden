package foo

type foo struct {
	bar string 
	t   bool   
	t   bool   
	qux string 
	yoo string 
}
